images/image.go

```diff
@@ -199,6 +205,49 @@ func Platforms(ctx context.Context, provider content.Provider, image ocispec.Des
 	}), ChildrenHandler(provider, "")), image)
 }
 
+// Check returns nil if the all components of an image are available in the
+// provider for the specified platform.
+//
+// If available is true, the caller can assume that required represents the
+// complete set of content required for the image.
+//
+// missing will have the components that are part of required but not avaiiable
+// in the provider.
+//
+// If there is a problem resolving content, an error will be returned.
+func Check(ctx context.Context, provider content.Provider, image ocispec.Descriptor, platform string) (available bool, required, present, missing []ocispec.Descriptor, err error) {
+	msft, err := Manifest(ctx, provider, image, platform)
```

Comment:
Microsoft? :sweat_smile: 

You may want to use a different variable name ;-)

Reply:
Na, this is the standard abbreviation for manifest. I don't think microsoft is a part of this context, so its not confusing.

@jhowardmsft "John Howard Manifest" Any objections to abbreviating Manifest to msft? :trollface:

Reply:
how is msft an abbreviation for manifest? The order of the letters do not match at all.

Reply:
LMAO. Yeah, mfst ;)